### PR TITLE
LIIKUNTA-55 | Add hero section

### PR DESCRIPTION
**Pull request by tuovinensanttu**
Added the hero section. The LandingPage type might change later, depending on how we will end up fetching the data.

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -6,28 +6,32 @@ import getPageData from "../components/page/getPageData";
...
-    ...recommendation,
-    keywords: recommendation.keywords.map((keyword) => ({
-      label: keyword,
-      onClick: () => {
-        router.push(`keywords/${encodeURIComponent(keyword)}`);
-      },
-      isHighlighted: keyword === "Maksuton",
-    })),
-  }));
+  const recommendationItems: Item[] = recommendations?.map(
```
Comment (tuovinensanttu) [inline src/pages/index.tsx:19]: Same thing here.

File: src/components/list/List.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ type Props = {
 function List({ items, component: Component }: Props) {
   return (
     <ul className={styles.list}>
-      {items.map((item) => (
+      {items?.map((item) => (
```
Comment (tuovinensanttu) [inline src/components/list/List.tsx:14]: This was the changed line. I needed the add the ? there, without it the application was crashing on startup.

File: src/components/hero/hero.module.scss [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    grid-column: 2;
+  }
+
+  @include breakpoints.respond-above(m) {
+    height: 40rem;
+  }
+
+  .box {
+    color: $color-white;
+    background-color: rgba($color-black, 80%);
```
Comment (Jokinen) [inline src/components/hero/hero.module.scss:26]: There's a `$color-black-80` variable in HDS you could also use.

File: src/components/hero/hero.module.scss [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    width: 20rem;
+    height: 15rem;
+    margin-top: var(--spacing-xl);
+    padding: var(--spacing-m);
```
Comment (Jokinen) [inline src/components/hero/hero.module.scss:30]: You've used scss variables elsewhere, but css variables here. Is there a reason for it?

File: src/components/hero/hero.module.scss [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+    @include breakpoints.respond-above(m) {
+      visibility: visible;
+    }
+  }
+
+  .block {
+    background-color: $color-white;
+    width: 100%;
+    height: calc(100% - 85px);
```
Comment (Jokinen) [inline src/components/hero/hero.module.scss:88]: What does this `85px` refer to?

File: src/components/hero/Hero.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+function Hero({ title, link, desktopImage }: LandingPage) {
+  return (
+    <div
+      className={styles.hero}
+      style={{ backgroundImage: `url(${desktopImage.uri})` }}
+    >
+      <div className={styles.box}>
+        <p>Kokosimme yhteen</p>
+        <h1>{title}</h1>
```
Comment (Jokinen) [inline src/components/hero/Hero.tsx:15]: Consider using https://github.com/City-of-Helsinki/liikunta-helsinki/tree/main/src/components/text for text components. That way they are consistently styled across pages.

File: src/components/hero/Hero.tsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+        <Link href={link}>
+          <Button className={styles.button}>Katso vinkit</Button>
```
Comment (Jokinen) [inline src/components/hero/Hero.tsx:17]: Would it be more appropriate to use an anchor element?

**Review by Jokinen — APPROVED** [review]
Code styles comments. Change looked good in the browser, good job 👍

File: src/components/hero/hero.module.scss (hunk 4 above)
Comment (tuovinensanttu) [reply src/components/hero/hero.module.scss:30]: Not really, constant context switching messes with my thinking. I'll fix these.

File: src/components/hero/hero.module.scss (hunk 5 above)
Comment (tuovinensanttu) [reply src/components/hero/hero.module.scss:88]: Forgot to add comment about this, the `85` is the default height `Koros` component has in `svg` and `rect`. By increasing one of these or both at the same didn't increase the filled height.

File: src/components/hero/Hero.tsx (hunk 6 above)
Comment (tuovinensanttu) [reply src/components/hero/Hero.tsx:15]: Will do, it skipped my mind this was already added.

File: src/components/hero/Hero.tsx (hunk 7 above)
Comment (tuovinensanttu) [reply src/components/hero/Hero.tsx:17]: I guess so, might require a little more styling.

**Pull request merged**